infra-as-code/bicep/modules/hubNetworking/generateddocs/hubNetworking.bicep.md

```diff
@@ -257,7 +257,7 @@ Azure Firewall Name.
 
 ![Parameter Setting](https://img.shields.io/badge/parameter-optional-green?style=flat-square)
 
-Switch to enable/disable Azure Firewall Policies deployment.
```

Comment:
I think we should keep this description as it was as it would be confusing for new deployments.